ansible-tests/validations/library/pacemaker.py

```diff
@@ -0,0 +1,42 @@
+import itertools
+from subprocess import Popen, PIPE
+from xml.etree import ElementTree
+
+from ansible.module_utils.basic import *
+
+
+def parse_pcs_status(pcs_status_xml):
+    root = ElementTree.fromstring(pcs_status_xml)
+    result = {
+        'failures': [dict(failure_element.items()) for failure_element
+                     in root.findall('failures/failure')],
+    }
+    return result
+
+
+def format_failure(failure):
```

Comment:
This method is a bit fragile and can potentially return oddly formatted strings if `pcs status` ever changes it's output format.

For extra safety, we should keep the keys of failure_element in parse_pcs_status above and use them to lookup items in your string interpolation below.